src/winforms/toga_winforms/widgets/box.py

```diff
@@ -24,5 +24,5 @@ def set_bounds(self, x, y, width, height):
             self.native.Location = Point(x - horizontal_shift, y + vertical_shift)
 
     def set_background_color(self, value):
-        new_color = native_color(value)
-        self.native.BackColor = new_color
+        if value:
+            self.native.BackColor = native_color(value)
```

Comment:
Why is the if clause needed here? Won't the default behavior of `native_color` catch that already? And if not... shouldn't it?

Reply:
The default behavior of `native_color` will do fine. I added the if clause because it can reduce overhead as this function is called with None as an argument many times (When color is not changed or set). And it looked like it is more consistent with the source code to change color in other platforms like iOS or macOS.